### DISCUSSION TITLE
Formula for subheader in  Big Number

### DISCUSSION THREAD

**Discussion by Riskatrih** [apache/superset#19181] [Q&A / Help]
i want to make a KPI like this 
![image](https://user-images.githubusercontent.com/59104917/111279320-28dc1000-866d-11eb-8d1d-f1bc33b6b1fa.png)
with visualization type 'Big Number With Trendline'. can supersets use a formula like MAX or MIN in the subheader to specify a value that is not a number?

**Comment by jetlag11235** [comment]
bumping.  we find ourselves copy/pasting charts and dashboards to support dynamic subheadings on Big Number charts.  echoing the original poster, the ability to set subheader using a custom metric would simplify our codebase.  e.g.,

```sql
format('from %s to %s', min(day), max(day))
```

we understand a table could be used to communicate this information, but prefer the Big Number visualization in some cases.  I think we are open to implementing this, but are looking for community feedback on the right feature to implement.
